### PR TITLE
Add TotW #74

### PR DESCRIPTION
**Pull request by manshreck**

### CODE REVIEW

File: _posts/2017-12-21-totw-74.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Originally posted as totw/74 on 2014-04-21
+
+By Bradley White [(bww@google.com)](mailto:bww@google.com)
+
+*"Delegating work works, provided the one delegating works, too." -- Robert
+Half*
+
+When a class has multiple constructors there is often a need to perform similar
+initialization in each variant. To avoid code duplication, many Google
```
Comment (tituswinters) [inline _posts/2017-12-21-totw-74.md]: many Google classes => many older classes?

File: _posts/2017-12-21-totw-74.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+Note that if you delegate to another constructor you cannot also use a member
+initialization list -- all initialization is done by the delegated constructor.
+Don’t go overboard though. If all the shared code does is set members, separate
+member-initialization lists or in-class initializers are probably still clearer.
+Use good judgement.
+
+<p class="note">
+Aside: This only matters if the constructors can throw, but the object is not
```
Comment (tituswinters) [inline _posts/2017-12-21-totw-74.md]: What is "this"?

File: _posts/2017-12-21-totw-74.md (hunk 1 above)
Comment (manshreck) [reply _posts/2017-12-21-totw-74.md]: Done

File: _posts/2017-12-21-totw-74.md (hunk 2 above)
Comment (manshreck) [reply _posts/2017-12-21-totw-74.md]: Reworded and added clarification. PTAL.

**Comment by manshreck** [comment]
Applied bww's recommendation on rewording.

**Pull request merged**